src/components/StarRating.tsx

```diff
@@ -7,27 +7,30 @@ interface Props {
 }
 
 export const StarRating = (props: Props) => {
+  const starSize = 16;
   const rating = _.min([5, props.rating]) || 0;
 
   return (
     <>
-      <StarDiv style={{ width: rating * 28 }} />
-      <HollowStarDiv style={{ width: 5 * 28 }} />
+      <StarDiv style={{ width: rating * starSize }} />
+      <HollowStarDiv style={{ width: 5 * starSize }} />
     </>
   );
 };
 
-const StarDiv = styled.div`
-  background-image: url(/starsolid.svg);
+const Star = styled.div`
   background-repeat: repeat-x;
-  height: 28px;
+  background-size: 16px;
+  height: 16px;
   position: absolute;
+`;
+
+const StarDiv = styled(Star)`
+  background-image: url(/starsolid.svg);
```

Comment:
Replace with imported image (images and other resource can be imported into js). Then background-image can be set to 

```js
 import star from "./starsolid.svg"

const StarDiv = styled(Star)`
  background-image: url(${star});
`;
```